setup.cfg

```diff
@@ -92,7 +93,7 @@ output-dir = invenio_administration/translations/
 add_ignore = D401
 
 [pycodestyle]
-max-line-length = 88
+ignore = E501
 
```

Comment:
I removed the max-line-length constraint because it clashes with Black formatting. 
If there are any better solutions, please provide input.

Reply:
you should be able to remove the whole `[pycodestyle]` part because it is not used anymore since we use black

Reply:
You mean to completely replace it in the `pytest.ini` cause If you don't explicitly ignore the `E501 line too long` pycodestyle will keep complaining. 
I updated `pytest.ini` to use black instead.

Reply:
oh there is still the `pytest.ini` file, this file has been removed from the other packages. further you could change the `pytest-black` dependency to `pytest-black-ng>=0.4.0`. `pytest-black` is unmaintained and `pytest-black-ng` fixes a warning.